src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
+  - name: Create a rule for http to https redirect
+    text: >
+        az cdn endpoint rule add -g group -n endpoint --profile-name profile --order 1\\
+            --rule-name "redirect" --match-variable RequestScheme --operator Equal --match-values HTTPS\\
+            --action-name "UrlRedirect" --redirect-protocol Https --redirect-type Moved
+"""
+
+helps['cdn endpoint rule remove'] = """
+type: command
+short-summary: Remove a delivery rule from an endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+examples:
+  - name: Remove the global rule.
+    text: >
+        az cdn endpoint rule remove -g group -n endpoint --profile-name profile --rule-name Global\\
+"""
+
+helps['cdn endpoint rule show'] = """
+type: command
+short-summary: show delivery rules asscociate with the endpoint.
+examples:
+  - name: show delivery rules asscociate with the endpoint.
+    text: >
+        az cdn endpoint rule show -g group --profile-name profile-name
+"""
+
+helps['cdn endpoint rule condition'] = """
+type: group
+short-summary: Manage delivery rule conditions for an endpoint.
+"""
+
+helps['cdn endpoint rule condition add'] = """
+type: command
+short-summary: Add a condition to a delivery rule.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --match-variable
+    type: string
+    short-summary: Name of the match condition.
+  - name: --operator
+    type: string
+    short-summary: Operator of the match condition.
+  - name: --selector
+    type: string
+    short-summary: Selector of the match condition.
+  - name: --match-values
+    type: string
+    short-summary: Match values of the match condition (comma separated).
+  - name: --negate-condition
+    type: string
+    short-summary: Describes if this is negate condition or not.
+  - name: --transform
+    type: string
+    short-summary: Describes what transforms are applied before matching. Possible values are Lowercase, Uppercase.
+examples:
+  - name: Add a remote address condition.
+    text: >
+        az cdn endpoint rule condition add -g group -n endpoint --profile-name profile --rule-name name\\
+            --match-variable RemoteAddress --operator GeoMatch --match-values "TH"
+"""
+
+helps['cdn endpoint rule condition remove'] = """
+type: command
+short-summary: Remove a condition from a delivery rule.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --index
+    type: string
+    short-summary: index of the condition.
+examples:
+  - name: Remove the first condition.
+    text: >
+        az cdn endpoint rule condition remove -g group -n endpoint --profile-name profile --rule-name name\\
+            --index 0
+"""
+
+helps['cdn endpoint rule condition show'] = """
+type: command
+short-summary: show delivery rules asscociate with the endpoint.
```

Comment:
can you make the  first letter of each sentence capitalized for all your summaries?